engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -106,6 +110,15 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   private val calls = workflowDescriptor.backendDescriptor.workflowNamespace.workflow.calls
 
+  // TODO: We should probably create a trait which loads all the configuration (once per application), and let classes mix it in
```

Comment:
Can you create a ticket for this - specifically the part about not calling `.load` everywhere although include your trait idea as an example (I like the idea, but don't want to specify that as the definitive solution at the moment)